tests/integration/targets/win_dns_zone/aliases

```diff
@@ -0,0 +1 @@
+shippable/windows/group1
```

Comment:
Please change to group 3. Based on the nightly run it is the group with the smallest runtime https://dev.azure.com/ansible/ansible.windows/_build/results?buildId=128352&view=results.

Reply:
Should we apply this change to all modules?